Sources/AblyAssetTrackingSubscriber/DefaultSubscriber.swift

```diff
@@ -54,6 +53,7 @@ class DefaultSubscriber: Subscriber {
         self.ablySubscriber.subscriberDelegate = self
 
         self.ablySubscriber.subscribeForAblyStateChange()
+        self.workerQueue.properties.subscriber = self
```

Comment:
As mentioned elsewhere, we can't make the worker queue’s properties public. We need to solve this in some other way.

How about the following:

1. make `workerQueue` an implicitly unwrapped optional, and then do the following two things inside `init`:
2. initialize the local var `properties` once `self` is available in `init`, passing it `subscriber: self`
3. then set up `self.workerQueue`

Reply:
A good suggestion, thanks. Perhaps not a biggest fan of implicitly force unwrapping in the property declaration, but in this case I'd say it's acceptable, considering the alternatives.
f0266d7194f0322a800dbafa0f31c35d2156fc63

Reply:
Yeah, this is the kind of situation that implicitly unwrapped optionals are intended for, I think — when you have two classes with references to each other and where one wants to create the other as part of its initialization.